android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/TranslationsBrowserState.kt

```diff
@@ -12,11 +13,13 @@ import mozilla.components.concept.engine.translate.TranslationSupport
  *
  * @property isEngineSupported Whether the translations engine supports the device architecture.
  * @property supportedLanguages Set of languages the translation engine supports.
+ * @property downloadedLanguages Set of languages the translation engine has downloaded.
```

Comment:
Would recommend changing this to `languageModels` and `Set of language machine learning translation models the translation engine has available.`

Reply:
Updated!